packages/luvi/build.sh

```diff
@@ -0,0 +1,52 @@
+TERMUX_PKG_HOMEPAGE=https://luvit.io
+TERMUX_PKG_DESCRIPTION="A project in-between luv and luvit."
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="Komo @cattokomo"
+TERMUX_PKG_VERSION=20240216
+_commit=b85da58f2549a519486a7296f5b836a8f6a64880
+_version=2.14.0
+TERMUX_PKG_GIT_BRANCH=master
+TERMUX_PKG_SRCURL=git+https://github.com/luvit/luvi.git
+TERMUX_PKG_DEPENDS="pcre, openssl, luv, lua51-lpeg, libluajit"
+TERMUX_PKG_BUILD_DEPENDS="luv, lua51-lpeg"
+TERMUX_PKG_SUGGESTS="lit, luvit"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+	 -DWithSharedLibluv=On
```

Comment:
Still some indent issues here.
```suggestion
	-DWithSharedLibluv=On
```